dali/pipeline/executor/executor.cc

```diff
@@ -436,6 +441,67 @@ void Executor<WorkspacePolicy, QueuePolicy>::RunHelper(OpNode &op_node, Workspac
 }
 
 
+template <typename WorkspacePolicy, typename QueuePolicy>
+void Executor<WorkspacePolicy, QueuePolicy>::ShareOutputsImpl(Workspace *ws, size_t iteration_id) {
+  DALI_ENFORCE(ws != nullptr, "Workspace is nullptr");
```

Comment:
This whole function has been copy-pasted from `executor.h`, with only the lines `470:471` added.